packages/jwt/lib/src/encoding.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:convert';
+
+/// Converts one of the three base64-encoded parts of a JWT to a JSON object.
+Map<String, dynamic> jwtPartToJson(String part) {
```

Comment:
```suggestion
Map<String, dynamic> jwtDecodePart(String part) {
```

nit feel free to ignore

Reply:
Went with `decodeJwtPart` ¯\_(ツ)_/¯